styles.css

```diff
@@ -0,0 +1,339 @@
+    width: 50%;
+  }
+  .card-desc{
+    padding:2px !important;
+  }
+  .card-desc p{
+    padding: 0px !important;
+    font-size: small !important;
+  }
+  footer{
+    width:100% !important;
+  }
+}
+/* Tablet Styles */
+@media only screen and (min-width:501px) and (max-width:959px){
+  header{
+    width:100%;
+  }
+  h4{
+    padding: 2px !important;
+  }
+  nav{
+    width: 100% !important;
+    padding: 5px;
+  }
+  body{
+    width: 100%;
+    background: lightpink;
+  }
+  .main-container{
+    margin: 3px !important;
+    grid-gap:5px !important;
+  }
+  section{
+    flex-wrap: wrap !important;
+    width:100% !important;
+  }
+  .section-left-side{
+    flex: 0% !important;
+  }
+  .section-left-side img{
+    height: 145px !important;
+  }
+  .section-right-side{
+    flex: 50% !important;
+    flex-wrap: wrap !important;
+    padding: 0px !important;
+  }
+  .section-right-side h3{
+    font-size: medium;
+  }
+  p{
+    padding: 0px !important;
+    font-size: small !important;
+    word-wrap: break-word !important;
+  }
+  article{
+    grid-gap:5px !important;
+    width: 100% !important;
+  }
+  .row-1{
+    grid-gap:5px !important;
+  }
+  .col{
+    width: 100%;
+  }
+  .card-desc{
+    padding:2px !important;
+  }
+  .card-desc p{
+    padding: 0px !important;
+    font-size: small !important;
+  }
+  .footer{
+    width:100%;
+  }
+}
+/* Desktop Styles */
+@media only screen and (min-width: 960px) {
+  body{
+    width:100%;
+  }
+  header{
+    width:100%;
+  }
+  .main-container{
+    width:70%;
+  }
+  footer{
+    width:100%;
+  }
+  .banner-left-side{
+    width:50%;
+  }
+  .banner-right-side{
+    width:50%;
+  }
+}
+.main-container{
+  display: grid;
+  grid-template-rows: auto auto auto;
+  grid-gap: 30px;
+  margin : 20px 150px;
+}
+h4,h5{
+  margin:0px;
+}
+h3{
+  margin:3px;
+}
+p{
+  margin:0;
+  font-weight: 100;
+  font-size: small;
+}
+a{
+  color:white;
+  text-decoration: none;
+}
+header{
+  display: flex;
+  background-image: url('images/header-image.jpg');
+  height: 250px;
+  padding:40px;
+}
+.banner-left-side{
+  flex:80%;
+}
+.banner-left-side-img{
+  height:75px;
+}
+.banner-right-side{
+  flex:5;
+}
+.banner-right-side-cart{
+  display:flex;
+  justify-content: space-between;
+  align-items :baseline;
+  width: 80px;
+  height:25px;
+}
+.cart{
+  height:20px;
+}
+.cart-price{
+  font-size: smaller;
+  color:white;
+  margin: 0;
+}
+nav{
+  background-color: #e2e5e6;
+  height : 50px;
+}
+.nav-caption{
+  color:darkblue;
+  font-size:13px;
+  padding : 20px 150px;
+  font-family: serif;
+}
+nav a{
+  color:darkblue;
+}
+section{
+  display:flex;
+}
+.section-left-side{
+  flex:30%;
+}
+.section-left-side img{
+  height:230px;
+}
+.section-right-side{
+  flex: 70%;
+  padding:0px 150px 0px 30px;
+}
+.section-right-side-header{
+  color:darkblue;
+  font-family: serif;
+  font-size:x-large;
+}
+.section-right-side-para{
+  font-family: monospace;
+  padding: 0px 19px 0px 0px;
+  font-size:larger;
```

Comment:
While this is responsive I am not a huge fan of using the keywords for font-size as it can vary from browser to browser. % and em is more consistent.